cmd/certificates-controller/OWNERS

```diff
@@ -0,0 +1,67 @@
+approvers:
```

Comment:
With the pending rename, please just put you, me, and mike as owners/approvers of this directory. Other folks can send PRs if they want to be added to the list.